supervisor/docker/homeassistant.py

```diff
@@ -109,23 +115,36 @@ def mounts(self) -> list[Mount]:
                     Mount(
                         type=MountType.BIND,
                         source=self.sys_config.path_extern_ssl.as_posix(),
-                        target="/ssl",
+                        target=PATH_SSL.as_posix(),
                         read_only=True,
                     ),
                     Mount(
                         type=MountType.BIND,
                         source=self.sys_config.path_extern_share.as_posix(),
-                        target="/share",
+                        target=PATH_SHARE.as_posix(),
                         read_only=False,
                         propagation=PropagationMode.RSLAVE.value,
                     ),
                     Mount(
                         type=MountType.BIND,
                         source=self.sys_config.path_extern_media.as_posix(),
-                        target="/media",
+                        target=PATH_MEDIA.as_posix(),
                         read_only=False,
                         propagation=PropagationMode.RSLAVE.value,
                     ),
+                    Mount(
+                        type=MountType.BIND,
+                        source=self.sys_config.path_extern_backup.as_posix(),
+                        target=PATH_BACKUP.as_posix(),
```

Comment:
Why is the bind mount for regular backup needed? And do we handle backups on external network storage as well?

Reply:
Because core will only use the location `.cloud_backup` for cloud-only backups. If the user wants it stored locally or in a network mount core will use that location instead. And we'll provide them with a path to the file within `/backup` to use for further processing.

I want it like this because of network mounts. I want to avoid writing backups to the local disk any time we can. So if the user wants a backup stored on their backup mount and google drive only then Supervisor will write it directly to the backup mount and core will read it out of there to upload it to google drive. No unnecessary writes on the local disk.